docs/operating-system/debugging.md

```diff
@@ -28,9 +28,11 @@ Make sure when you are copying the public key to the root of the USB drive that
 You should then be able to SSH into your Home Assistant device. On Mac/Linux, use:
 
 ```shell
-ssh root@hassio.local -p 22222
+ssh root@homeassistant.local -p 22222
 ```
 
+Older installations may need to use `ssh root@hassio.local -p 22222`. You can check the correct hostname to use in the System page of the Supervisor interface in Home Assistant.
```

Comment:
```suggestion
Older installations may need to use `ssh root@homeassistant.local -p 22222`. You can check the correct hostname to use in the System page of the Supervisor interface in Home Assistant.
```

Reply:
This would make the example here identical to the one above. I was trying to cover the case of old setups still having the `hassio` hostname. Maybe just leave out the alternative example and point to checking the hostname?

Reply:
Maybe make it more generic, something like:

> If you have an older instance or changed your hostname, you might need to use a different hostname to connect to. You can find the correct hostname to use in the System page of the Supervisor interface in Home Assistant.